template/src/pageComponents/mint/steps/OutOfGasStep/OutOfGasStep.tsx

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
+import clsx from 'clsx';
+import { MintSteps } from '../../ContractDemo';
+
```

Comment:
Open question do we want all this levels?
`steps/OutOfGasStep/OutOfGasStep.tsx`

or just have all steps file under `steps/OutOfGasStep.tsx`

Reply:
That's a good point. Initially I added them under their own folders in case we needed other files there, but we can have this more top-level. I will make that change.

Reply:
My point in general with BOAT, is to keep things as flat as possible, as than the developer that will eventually use it, can decide how to re-organizite after.